### ISSUE TITLE
Discrepancy with Nightingales - Geoduck RRBS

### ISSUE THREAD

**Issue by sr320** [RobertsLab/resources#258]
In the spreadsheet 
https://docs.google.com/spreadsheets/d/1_XqIOPVHSBVGscnjzDSWUeRL7HUHXfaHxVzec-I-8Xk/edit#gid=0

there are 50 "EPI" samples listed. 
These are the RRBS libraries Hollie made.

on Owl
`/Volumes/web/nightingales/P_generosa`

there are 106 files corresponding to 53 libraries as they are paired end.

What is the discrepancy?

**Comment by kubu4** [comment]
Fixed discrepancies:

- Fixed typo in nightingales sheet
- Added three unlisted files to nightingales sheet

Fixed:
- file path listings (added concatenation formula) in nightingales sheet (just these geoduck files)
- URL (added hyperlink formula) are now clickable links

I believe issues are likely related to the file merger that Sean implemented a while back. Regardless, all files in ```owl/nightingales/P_generosa/EPI*``` are now listed in nightingales sheet with proper paths/URLs.

However, I do not know how ```Library_name``` was assigned, so I have not added that info to the three "new" files I added to nightingales sheet.
